src/Mod/Mesh/Gui/Command.cpp

```diff
@@ -952,9 +952,9 @@ CmdMeshCrossSections::CmdMeshCrossSections()
 {
     sAppModule = "Mesh";
     sGroup = QT_TR_NOOP("Mesh");
-    sMenuText = QT_TR_NOOP("Cross-sections...");
-    sToolTipText = QT_TR_NOOP("Cross-sections");
-    sStatusTip = QT_TR_NOOP("Cross-sections");
+    sMenuText = QT_TR_NOOP("Cross-Sections…");
+    sToolTipText = QT_TR_NOOP("Cross-sections the mesh");

```

Comment:
```suggestion
    sToolTipText = QT_TR_NOOP("Creates cross-sections of the mesh");
```
https://wiki.freecad.org/Mesh_CrossSections